src/virtac/atip_server.py

```diff
@@ -250,13 +254,17 @@ def _on_update(self, value, name):
         """The callback function passed to out records, it is called after
         successful record processing has been completed. It updates the out
         record's corresponding in record with the value that has been set and
-        then sets the value to the centralised Pytac lattice.
+        then sets the value to the centralised Pytac lattice. This functions
+        needs to be kept FAST as it can be called quickly by CA clients.
```

Comment:
```suggestion
        then sets the value to the centralised Pytac lattice.
        
        This function needs to be kept FAST as it needs to handle frequent CA updates.
```

Reply:
Not sure what 'sets the value to the centralised Pytac lattice' is supposed to mean. @T-Nicholls , any ideas to improve this?

Reply:
It just means that it calls `set_value` on the relevant element in the Pytac lattice.
Maybe just remove the word "centralised"?

Reply:
@ptsOSL, you did not see my original suggestion to move the second sentence down and improve wording.